infoschema/metric_table_def.go

```diff
@@ -16,6 +16,16 @@ package infoschema
 // MetricTableMap records the metric table definition, export for test.
 // TODO: read from system table.
 var MetricTableMap = map[string]MetricTableDef{
+	"tidb_ops_total": {
+		PromQL:  `sum(tidb_executor_statement_total{$LABEL_CONDITIONS}) by (instance)`,
+		Labels:  []string{"instance"},
+		Comment: "TiDB SQL total",
+	},
+	"tidb_ops_type": {
+		PromQL:  `sum(tidb_executor_statement_total{$LABEL_CONDITIONS}) by (instance,type)`,
+		Labels:  []string{"instance", "type"},
+		Comment: "TiDB SQL total",
```

Comment:
Please use a more detail comment such like `TiDB SQL total by instance and type`